nested_open_struct.gemspec

```diff
@@ -6,7 +6,10 @@ Gem::Specification.new do |s|
   s.homepage    = "http://github.com/porras/nested_open_struct"
   s.summary     = "Thin wrapper around OpenStruct to make structs 'nested'"
   s.description = "Thin wrapper around OpenStruct to make structs 'nested'"
- 
+
   s.files        = Dir.glob("lib/**/*") + %w(LICENSE README.md)
   s.require_path = 'lib'
-end
\ No newline at end of file
+
+  s.add_development_dependency 'minitest', '~> 5.8'
+  s.add_development_dependency 'rake',     '~> 10.4'
```

Comment:
Features of Gemfile I was more thinking of was the ability to use gem dependencies (for dev) from disk or github. Also for shoes4 it's nice where we have a multi gem repository and we have the dev dependencies dried up in one Gemfile at the top level and then the runtime dependencies in each gemspec. 

Anyhow, matter of test :)

Also yes, :+1: no Gemfile.lock for libraries, Gemfile.lock is for applications :)